eva_assembly_ingestion/nextflow/remap_cluster.nf

```diff
@@ -0,0 +1,279 @@
+    $params.executable.nextflow run $params.nextflow.remapping -resume \
+      --oldgenome `pwd`/${source_fasta} \
+      --newgenome `pwd`/${target_fasta} \
+      --vcffile `pwd`/${source_vcf} \
+      --outfile `pwd`/${basename_source_vcf}_remapped.vcf
+    """
+}
+
+
+/*
+ * Ingest the remapped submitted variants from a VCF file into the accessioning warehouse.
+ */
+process ingest_vcf_into_mongo {
+    memory "${params.memory}GB"
+    clusterOptions "-g /accession"
+
+    input:
+    path remapped_vcf from remapped_vcfs.flatten()
+    path target_report from updated_target_report
+
+    output:
+    path "${remapped_vcf}_ingestion.log" into ingestion_log_filename
+
+    publishDir "$params.output_dir/logs", overwrite: true, mode: "copy", pattern: "*.log*"
+
+    script:
+    """
+    # Check the file name to know which database to load the variants into
+    if [[ $remapped_vcf == *_eva_remapped.vcf ]]
+    then
+        loadTo=EVA
+    else
+        loadTo=DBSNP
+    fi
+
+    java -Xmx8G -jar $params.jar.vcf_ingestion \
+        --spring.config.name=${params.ingestion_properties} \
+        --parameters.vcf=${remapped_vcf} \
+        --parameters.assemblyReportUrl=file:${target_report} \
+        --parameters.loadTo=\${loadTo} \
+        > ${remapped_vcf}_ingestion.log
+    """
+}
+
+process process_remapped_variants {
+    memory "${params.memory}GB"
+    clusterOptions "-g /accession"
+
+    input:
+    path ingestion_log from empty_ch.mix(ingestion_log_filename.collect())
+
+    output:
+    path "${source_to_target}_process_remapped.log" into process_remapped_log_filename
+    // TODO this also generates a rs report, for "newly remapped" rs - should we QC this separately?
+    // path "${source_to_target}_rs_report.txt" optional true into rs_report_filename
+
+    publishDir "$params.output_dir/logs", overwrite: true, mode: "copy", pattern: "*.log*"
+
+    """
+    java -Xmx8G -jar $params.jar.clustering \
+        --spring.config.name=${params.clustering_properties} \
+        --spring.batch.job.names=PROCESS_REMAPPED_VARIANTS_WITH_RS_JOB \
+        > ${source_to_target}_process_remapped.log
+    """
+}
+
+process cluster_unclustered_variants {
+    memory "${params.memory}GB"
+    clusterOptions "-g /accession/instance-${params.clustering_instance}"
+
+    input:
+    path process_remapped_log from process_remapped_log_filename
+
+    output:
+    path "${source_to_target}_clustering.log" into clustering_log_filename
+    path "${source_to_target}_rs_report.txt" optional true into rs_report_filename
+
+    publishDir "$params.output_dir/logs", overwrite: true, mode: "copy", pattern: "*.log*"
+
+    """
+    java -Xmx8G -jar $params.jar.clustering \
+        --spring.config.name=${params.clustering_properties} \
+        --spring.batch.job.names=CLUSTER_UNCLUSTERED_VARIANTS_JOB \
+        > ${source_to_target}_clustering.log
+    """
+}
+
+/*
+ * Run clustering QC job
+ */
+process qc_clustering {
+    memory "${params.memory}GB"
```

Comment:
Suggest adding `clusterOptions "-g /accession` to enforce connection limits to job tracker